### DISCUSSION TITLE
startupOptions(StartupOptions.initial())时无任何输出

### DISCUSSION THREAD

**Discussion by kpretty** [apache/flink-cdc#2141] [Q&A]
之前项目的环境是 flink 1.12 + flink-connector-mysql-cdc 2.0.0 使用 `startupOptions(StartupOptions.initial())`启动模式是可以获取到数据库最新的快照数据的
![image](https://github.com/ververica/flink-cdc-connectors/assets/77819741/a286c899-b987-463f-822a-333035356644)
版本升级后 flink 1.17 +  flink-connector-mysql-cdc 2.3.0 同样使用 `startupOptions(StartupOptions.initial())` 启动模式居然无任何输出，这可能是因为什么
<img width="1366" alt="image" src="https://github.com/ververica/flink-cdc-connectors/assets/77819741/9524c45a-dea0-4996-99b5-7d16abcff9a2">

**Comment by kpretty** [comment]
尝试开启 checkpoint 并没有解决问题

**Reply by yangleiRandium** [reply]
最后解决了吗。我也遇到同样的问题。不知道怎么排查
